corehq/apps/reports/tests/test_util.py

```diff
@@ -187,7 +187,7 @@ def _send_case_to_es(
     ):
         case_json = {}
         if is_copy:
-            case_json[COMMCARE_CASE_COPY_PROPERTY_NAME] = 'True'
+            case_json[CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAMfE] = 'True'
```

Comment:
I think the value here also needs to be changed to a case ID.

Reply:
@zandre-eng Updated now. I just updated it to `case_id`, because the value isn't important and in the test I don't have/need the original case, so I just simply set it to `case_id` to make it clear that's what the value is normally, instead of the boolean.